app/src/main/java/com/yacgroup/yacguide/utils/MimeTypeMap.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022 Christian Sommer
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.yacgroup.yacguide.utils
+
+/*
+ * Normally this function is provided by library android.webkit.MimeTypeMap.
+ * The library seems to have a bug in API 26 and 27 (and maybe in other versions),
+ * so that the file extension cannot be determined by MIME type "application/json".
+ */
+enum class MimeTypeMap(val mimeType: String) {
```

Comment:
Please do not duplicate code. There is already the enum class `TourbookExportFormat`. Add the mime type there:
```
enum class TourbookExportFormat(val id: Int, val extension: String, val mimeType: String) {
    eJSON(0, "json", "application/json"),
    ...
}
```